assets/queries/cloudFormation/ecs_task_defenition_invalid_cpu_or_memory/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ECS Task_defenition_invalid_cpu_or_memory",
+  "queryName": "ECS Task Defenition Invalid Cpu or Memory",
```

Comment:
Fix typo and missing underscore.
"id": "ECS_Task_definition_invalid_cpu_or_memory",
  "queryName": "ECS Task Definition Invalid Cpu or Memory",